### PR TITLE
Working version

### PR DESCRIPTION
**Pull request by alexjreich**

### CODE REVIEW

File: src/controllers/apiController.js [hunk 1]
```diff
@@ -30,6 +34,65 @@ export default class APIController {
...
+  }
+
+  _saveProfile(req, res) {
+    var user = req.body
+    user.enabled = true
+    if(!user.admin)
+      user.admin = false
+    if(user.id == "")
+      delete user.id
+    console.log('user.password', user.password)
```
Comment (loppear) [inline src/controllers/apiController.js:48]: Probably should remove these logs with values.

**Comment by loppear** [comment]
Other than log comment, :+1:

**Comment by loppear** [comment]
So looking at the integration PR, I assume this purposely does not implement `GET /forgot` so that the including application can define that template... that's fine for now, but another case for figuring out how applications could selectively override templates and views routes, whether here or admin or .. would be nice if this could provide a default implementation etc, and the application could a) just provide an alternate template partial or b) fully replace the route with precedence over the default implementation.

And another case, so that the application could override the email template. With documentation of template names, presumably an application could `app.get('templater').register('user-forgot-password', 'ejs', 'path/to/specific.ejs')` so long as that register was `after` the normal gather of these. Might want to wrap that still, but that works for me for (a).

**Comment by alexjreich** [comment]
@loppear yeah, the plumbing in there is ready for modules to define routes, with the expectation that there is a template using the 'default' namespace defined by the app.  See _profileHandler in apiController.js.

**Pull request merged**